### DISCUSSION TITLE
Running code in quarto and Ipynb fails in codespaces

### DISCUSSION THREAD

**Discussion by Robinlovelace** [community/community#34528] [Codespaces]
This repo opens fine in GitHub [Codespaces](https://github.com/features/codespaces): https://github.com/geocompr/py 

However, the interactive Python environment fails. It just hangs, as shown in below.

[Screencast from 30-09-22 21:38:59.webm](https://user-images.githubusercontent.com/1825120/193353045-a4f7a14b-2273-4d5c-8711-cfcce6abd716.webm)

Reproducible example, try clicking here: https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=447558863

Note: you may need to have registered for GitHub codespaces for this to work.

**Comment by jfullstackdev** [comment]
if you have the public version or just a temp .ipynb file, you can send the link, so I can take a closer look also

**Reply by Robinlovelace** [reply]
Sure, here's the repo: https://github.com/geocompr/py you should be able to see the repo from that link but clearly not. Any further info or way I can help with this let me know. Thank you!

**Comment by jfullstackdev** [comment]
I can open the files without much loading

![image](https://user-images.githubusercontent.com/47092464/193299548-5b84e86c-df33-4fb9-b888-eb5ca9b1c045.png)

**Reply by Robinlovelace** [reply]
I'm talking about running code in GitHub codespaces, not opening the file. See here for more on Codespaces: https://github.com/features/codespaces

**Reply by Robinlovelace** [reply]
I will update the issue to provide more details.

**Reply by jfullstackdev** [reply]
yup, I'm using Codespaces too, but never had the same ipynb
